src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1315,6 +1315,7 @@ private static bool IsMemberDeclaration(SyntaxNode node)
                 case SyntaxKind.PropertyDeclaration:
                 case SyntaxKind.GetAccessorDeclaration:
                 case SyntaxKind.SetAccessorDeclaration:
+                case SyntaxKind.InitAccessorDeclaration:
```

Comment:
@davidwengier I removed both GetAccessorDeclaration and InitAccessorDeclaration in a draft. There are no test failures. So the current tests, unfortunately, doesn't cover those.

The usages that depend on this method are in:

- CodeLens\CodeLensReferencesService.cs (there is a call to `syntaxFactsService.IsDeclaration(node)`)
- CodeRefactorings\SyncNamespace\AbstractChangeNamespaceService.cs (there is a call to `.DescendantNodes(n => !syntaxFacts.IsDeclaration(n))`)

Unlike the other change, I'm not able to guess the effect of this change.
I'll try to change the method to only `return false` in the draft in a hope to see test failures (i.e. other switch arms are tested).